R/pdf_document.R

```diff
@@ -109,6 +109,7 @@ pdf_document <- function(toc = FALSE,
   # template path and assets
   if (identical(template, "default")) {
 
+    if (!pandoc_available()) stop("Pandoc not found")
```

Comment:
is it possible to say where we looked for pandoc to make it more obvious to the user that they have something misconfigured?